examples/catalog/lib/custom_a11y_traversal.dart

```diff
@@ -98,7 +98,8 @@ class SpinnerButton extends StatelessWidget {
       this.rowOrder,
       this.columnOrder,
       this.field,
-      this.increment}) : super(key: key);
+      this.increment})
```

Comment:
This should be:
```
    ...
    this.increment,
  }) : super(key: key);
```

Reply:
Note that this sort of fix is a sign switching to the formatter for this code could succeed. These cases we just need to manually add the missing commas.